rasa/cli/test.py

```diff
@@ -223,8 +225,8 @@ async def run_nlu_test_async(
         )
     elif cross_validation:
         logger.info("Test model using cross validation.")
-        config = rasa.cli.utils.get_validated_path(
-            config, "config", DEFAULT_CONFIG_PATH
+        config = str(
```

Comment:
yeah I tried to do that, but there was a lot of code to change. And given the size of the PR, I decided against it. I'll add a FIXME though, as you're right, this would be the proper fix

Reply:
fine